new-dti-website/tailwind.config.ts

```diff
@@ -62,7 +62,9 @@ module.exports = {
         card: {
           DEFAULT: 'hsl(var(--card))',
           foreground: 'hsl(var(--card-foreground))'
-        }
```

Comment:
@JasonMun7 @cchrischen @Bookie0 I wanted to define primary and secondary colors for the hero pages here to reuse. LMK if this isn't the most idiomatic/doesn't conform to the system you've been using though!